lib/berkeley_library/docker/secret.rb

```diff
@@ -1,36 +1,59 @@
 require 'berkeley_library/docker/logging'
+require 'digest'
+require 'time'
 
 module BerkeleyLibrary
   module Docker
     class Secret
       class << self
         include Logging
 
-        PATH_OVERRIDE_ENVVAR = 'UCBLIB_SECRETS_PATH'
-        DEFAULT_SECRETS_PATH = '/run/secrets'
+        PATH_OVERRIDE_ENVVAR = 'UCBLIB_SECRETS_PATTERN'
```

Comment:
I chose to rename this for clarity, since it's actually more like a globbing pattern than a path.